### ISSUE TITLE
error in first steps

### ISSUE THREAD

**Issue by gshirazinejad** [comet-licsar/licsbas#59]
Hi, I constantly get these errors when I want to run licsbas. I would appreciate to help me. errors are:

1- libcomcat is not installed. Identification of earthquake offsets will not work. How can I install 'libcomcat'?
2- In step 02 I always get these errors: 
           /home/gshirazinejad/miniconda3/envs/comet_licsbas/lib/python3.12/site-packages/osgeo/gdal.py:311: FutureWarning: Neither gdal.UseExceptions() nor gdal.DontUseExceptions() has been explicitly called. In GDAL 4.0, exceptions will be enabled by default.
  warnings.warn(      
           
/home/gshirazinejad/licsbas/comet_licsbas/LiCSBAS/bin/LiCSBAS02_ml_prep.py:480: RuntimeWarning: invalid value encountered in cast
  cc = cc.astype(np.uint8) ##nan->0, max255, auto-floored

![image](https://github.com/user-attachments/assets/9c69d92b-5022-4514-9da1-8f6d06b578e5)

3- I get this error in step 03: 
/home/gshirazinejad/miniconda3/envs/comet_licsbas/lib/python3.12/site-packages/osgeo/gdal.py:311: FutureWarning: Neither gdal.UseExceptions() nor gdal.DontUseExceptions() has been explicitly called. In GDAL 4.0, exceptions will be enabled by default.
  warnings.warn(

![image](https://github.com/user-attachments/assets/1b3044a0-f07d-4e36-bec9-4b1b167b4bf7)

4- I get these errors in step 12 and the proccesing is unsuccesfull:
![image](https://github.com/user-attachments/assets/6c51faf2-fcbe-42e2-9298-46cb4c9d46cb)
![image](https://github.com/user-attachments/assets/455beb6d-ca89-4281-966f-0276a73a203e)
![image](https://github.com/user-attachments/assets/c2ad9f1b-f10c-41cb-b212-317baf5d63bf)

**Comment by espiritocz** [comment]
hi. please differentiate between Warning and ERROR. Most of what you report as 'ERROR' are actually only (annoying) warnings. These help us figure out sources of issues.
But thank you for reporting the error regarding colour map - I believe you did not have paths set up properly (running LiCSBAS_bash.sh would set the paths in given terminal. If to keep it for other terminal, you may source this via your ~/.bashrc as stated in the Installation instructions. However, we updated the colour map settings with the latest release - please install the latest release (or do git fetch) and check the additional requirements - we added cmcrameri as requirement that will overcome the issue with paths.

**Comment by gshirazinejad** [comment]
Thank you for response. I didn't know that they are warnings...sorry.
About my first question, How can I install 'libcomcat'?
I always source the codes with  ~/bashrc_LiCSBAS.sh file but I got that warning. You said I did not have paths set up properly, You mean it is better to run codes with batch_LiCSBAS.sh file?
I am working with latest release and I have installed cmcrameri, however I got the error regarding colour map.
